ProjectShop/Shipping/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from django_countries.fields import CountryField
+
+class ShippingModel(models.Model):
+    # shipping_address_id = models.IntegerField(verbose_name="shipping address Id", primary_key=True)
+    postal_code = models.CharField(verbose_name="postal code", null=False, blank=False, max_length=20)
+    country = CountryField()
+    region = models.CharField(verbose_name="region", null=False, blank=False, max_length=50)
+    city = models.CharField(verbose_name="city", null=False, blank=False, max_length=50)
+    post_office = models.IntegerField(verbose_name="post office", null=False, blank=False)
```

Comment:
It would be good to add a translation.
Please, check this link for more information:
https://docs.djangoproject.com/en/3.2/topics/i18n/translation/